pkg/cluster/cluster.go

```diff
@@ -443,6 +449,30 @@ func (os *ClusterSpec) Validate() error {
 	return nil
 }
 
+func (s *ClusterSpec) validateReplicationSlots() error {
+	if len(s.AdditionalReplicationSlotNames) == 0 {
+		return nil
+	}
+
+	rp := regexp.MustCompile("[_0-9a-z]+")
```

Comment:
put it in a global var like done here: https://github.com/sorintlab/stolon/blob/master/pkg/postgresql/utils.go#L39